src/packet_normal.c

```diff
@@ -0,0 +1,84 @@
+#include "packet_normal.h"
+#include "errors.h"
+#include "message.h"
+#include "packet.h"
+
+PacketNormal *decode_normal(uint8_t *buf, size_t len, PacketHeader header, Error *err) {
+	PacketNormal *packet = malloc(sizeof(PacketNormal));
+	packet->_ = PACKET_NORMAL;
+	packet->header = header;
+
+	uint8_t *end = buf + len;
+	int32_t num_chunks = 0;
```

Comment:
Use `size_t`

Reply:
Oh yea true. I do not have the habit of using anything but int :D
Compiler warnings and casting have trained me to go int everywhere and live an easy life. But it indeed seems more correct conceptually.

I noticed the packet header uses ``uint8_t`` so I went with that for consistency.

https://github.com/MilkeeyCat/ddnet_protocol/blob/3a1d4c4403b9050d8deb8ce5faffcc87347b0a40/include/ddnet_protocol/packet.h#L38